ibm/service/kubernetes/resource_ibm_container_alb_create.go

```diff
@@ -91,11 +96,13 @@ func ResourceIBMContainerAlbCreate() *schema.Resource {
 				Type:        schema.TypeString,
 				Computed:    true,
 				Description: "ALB name",
+				Deprecated:  "Remove this attribute's configuration as it no longer is used",
```

Comment:
There are some Deprecated descriptions in this file that should be also updated to be consistent.